compatibility/bazel_tools/daml_ledger/Sandbox.hs

```diff
@@ -143,8 +143,8 @@ nullDevice
     | otherwise =  "/dev/null"
 
 readPortFile :: Int -> String -> IO Int
-readPortFile 0 _file = do
-  T.hPutStrLn stderr "Port file was not written to in time."
+readPortFile 0 file = do
+  T.hPutStrLn stderr ("Port file was not written to '" ++ file ++ "' in time.")
```

Comment:
This fails to build because `file :: String`, but `T.hPutStrLn` expects `Text`. You could use [`pack`](https://hackage.haskell.org/package/text-1.2.5.0/docs/Data-Text.html#v:pack) in something like this instead:
```
"Port file ..." <> T.pack file <> "..."
```